docs/datamodels/source/devel/attributes.rst

```diff
@@ -61,26 +68,29 @@ is stored in a separate image extension).  Full-frame data for all other
 instruments have four columns or rows of reference pixels on each edge
 of the image.
 
-Model classes
--------------
-
-Base class
-''''''''''
+DataModel Base Class
+--------------------
 
 .. autoclass:: jwst.datamodels.DataModel
    :members:
 
-Concrete model classes
-''''''''''''''''''''''
+Specific Model Classes
+----------------------
```

Comment:
It's the same list, but it uses the Sphinx `automodule` to generate the docstring documentation for each DataModel class.  The list at the top merely serves as a link to each chunk of documentation at the bottom.